### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by moeffju** [lukakerr/Pine#91]
I stumbled upon this from a web search and was pretty happy reading the features list, but it looks like development has stagnated. Is there ongoing development or plans for it?

**Comment by lukakerr** [comment]
Yep still alive. I'm in my final term of university at the moment and writing a thesis so very busy and haven't been able to work on Pine at all. I plan to continue development once I graduate

**Comment by moeffju** [comment]
Great, thanks for the feedback!

If you or one of the other contributors have the time, are there any bugs that would be good for a new contributor to try and tackle?

**Comment by lukakerr** [comment]
There aren't any other contributors really, it's just me 😅, but a few issues that may be easy to start with are #89, #83, #77

**Comment by r-bliss** [comment]
I concur—I've been following this project since last year and have been really interested in it. It really suits what I'm looking for in a markdown editor, so I'm hopeful development picks back up, especially as Macdown seemingly has ebbed.

**Comment by jordanlambrecht** [comment]
@lukakerr How about now? Still kickin'?

**Comment by SteveShi** [comment]
It's been five years now, but I still hope the developers will come back.

**Comment by khurshid-alam** [comment]
There is no native build available anywhere for apple silicon. This project is as good as dead.
